csm/common/errors.py

```diff
@@ -48,8 +58,9 @@ class CsmSetupError(CsmError):
     """
     _desc = "Csm Setup is failed"
```

Comment:
Csm Setup is failed -> CSM setup failed.

Reply:
updated